packages/components/checkbox/src/Checkbox.tsx

```diff
@@ -79,6 +79,12 @@ export const Checkbox = forwardRef<HTMLButtonElement, CheckboxProps>(
       checkboxIntent: intentProp,
     })
 
+    const getIsRequired = () => {
```

Comment:
Wdyt of using a memoized const instead? It might be more understandable code readers, and  would also take care of possible multiple toggles within a checkbox group :+1:

Reply:
I'm not sure I understand, you mean something like this?:
```tsx
const isRequired = useMemo(() => {
  if (!group) return isRequired

  return isRequired ? !group.value?.length : false
}, [group, isRequired])
```

Reply:
Yes!